src/sysgpu/d3d12.zig

```diff
@@ -84,11 +84,28 @@ pub const Instance = struct {
             &c.IID_IDXGIFactory4,
             @ptrCast(&dxgi_factory),
         );
-        if (hr != c.S_OK) {
+        if (hr == c.DXGI_ERROR_INVALID_CALL) {
+            const hr_prev = hr;
+            hr = c.CreateDXGIFactory2(
+                0,
+                &c.IID_IDXGIFactory4,
+                @ptrCast(&dxgi_factory));
+            if (hr == c.S_OK) {
+                std.debug.print(
+                    \\Failed to enable debug layer ({x}):
+                         \\ For Windows 10, to create a device that supports the debug layer, enable the "Graphics Tools" optional
+                         \\ feature. On Windows 10 22H2, go to the Settings panel, under System. On older versions of Windows 10,
+                         \\ go to the Settings panel, under Apps > Apps & features. Go to Optional Features > Add a feature, and
+                         \\ then look for "Graphics Tools"
+                         \\                    
```

Comment:
1. Can you use `log.info` instead of `std.debug.print`? That lets others silence this warning if they want, etc. and is more consistent with what we do elsewhere in this file.

2. Could you also add a link to the Microsoft article here at the end? In case this info becomes out of data it'd be useful probably.

Reply:
Actually, thinking about it a bit more, maybe let's go with a shorter and simpler message using `log.info`:

> note: D3D12 debug layers disabled (couldn't enable, error: {x}), see https://machengine.org/about/faq/#how-to-enable-direct3d-debug-layers

And then I can add this info to the FAQ on the website.